docs/book/cookbook/aot-guide.md

```diff
@@ -23,10 +23,11 @@ composer create-project zendframework/zend-expressive-skeleton zend-di-aot-examp
 Pick the components you want to use. As statet above, we will be using Zend ServiceManager
 and a Modular layout for this example.
 
-Now add zend-di with composer:
+Now add zend-di and zend-code with composer:
```

Comment:
At this point it is not clear why zend-code is needed. Please add an explanation in the sentence above.